ae-ease-to-gsap-customease.jsx

```diff
@@ -248,25 +248,15 @@
 		var endInterpolation = property.keyInInterpolationType(endIndex);
 
 		if (startInterpolation === KeyframeInterpolationType.LINEAR &&
-			endInterpolation === KeyframeInterpolationType.LINEAR) {
+			endInterpolation !== KeyframeInterpolationType.HOLD) {
```

Comment:
My bad, I didn't read the return values properly, they looked the same at a glance, so I assumed they were.
I'll add a revert of this commit